keyboards/nopunin10did/jabberwocky/v2/readme.md

```diff
@@ -0,0 +1,35 @@
+# Jabberwocky Gen. 2
+
+![Jabberwocky v2 CNC Edition](https://diplomacyvariants.files.wordpress.com/2022/10/img_5151.jpg)
```

Comment:
please rehost this on imgur (or the like).

Reply:
I've updated this URL.